api/sys_plugins.go

```diff
@@ -299,6 +302,13 @@ func (c *Sys) ReloadPluginStatus(reloadStatusInput *ReloadPluginStatusInput) (*R
 
 	resp, err := c.c.RawRequestWithContext(ctx, req)
 	if err != nil {
+		defer resp.Body.Close()
```

Comment:
why not just have a single `defer resp.Body.Close()` call?

Reply:
No reason!  Fixed.